cli/utils.go

```diff
@@ -248,3 +250,7 @@ func runServices(ctx context.Context, makeServices ...func(context.Context) (jas
 	<-ctx.Done()
 	return closeAllServices(closeServices)
 }
+
+func randDur(window time.Duration) time.Duration {
+	return window + time.Duration(rand.Int63n(int64(window)))
```

Comment:
This returns a number in [n, 2n), not [n/2, 3n/2). Is that deliberate?

Reply:
yeah.